packages/sdk/src/polkadot/PolkadotService.ts

```diff
@@ -152,9 +152,9 @@ export class PolkadotService {
   async getAssetDecimals(asset: Asset): Promise<number> {
     const metaId = this.chain.getMetadataAssetId(asset);
     return (
+      this.chain.getAssetDecimals(asset) ||
```

Comment:
I feel like the `decimals` we provide in the config should override all the rest. I'm open to comments